cg/meta/archive/ddn/ddn_data_flow_client.py

```diff
@@ -202,7 +209,58 @@ def delete_file(self, file_and_sample: FileAndSample) -> None:
         delete_file_payload = DeleteFilePayload(
             global_path=f"{self.archive_repository}{sample_id}/{file_name}"
         )
-        delete_file_payload.delete_file(
-            url=urljoin(self.url, DataflowEndpoints.DELETE_FILE),
+        response: Response = self._post_request(
+            endpoint=DataflowEndpoints.DELETE_FILE,
             headers=dict(self.headers, **self.auth_header),
```

Comment:
Would we want this dict as a class variable? e.g. self.authorized_headers or something?

Reply:
is it used in many other places?

Reply:
It is used in all posts except for when we fetch authorization tokens in the beginning.

Reply:
In another PR if that's the case